src/main/java/uk/gov/companieshouse/overseasentitiesapi/utils/Constants.java

```diff
@@ -50,5 +50,5 @@ private Constants() {
 
     public static final String CONCATENATED_STRING_FORMAT = "%s,%s";
     public static final String CONCATENATED_STRING_FORMAT_NO_COMMA = "%s%s";
-    public static final SchemaVersion CURRENT_MONGO_SCHEMA_VERSION = SchemaVersion.VERSION_3_1;
+    public static final SchemaVersion CURRENT_MONGO_SCHEMA_VERSION = SchemaVersion.VERSION_3_2;
```

Comment:
So we need to change the version here? I thought all that was to do with the entity name field.

Reply:
We don't _have_ to change the version, because the addition of new fields to the JSON/data model is not a breaking change. The entity name field changes involved converting a new string field (for entity name) into a structure that contained an entity name and that broke with older versions of the JSON.

Nevertheless, good to define a new version at this point I think, as it will identify any new JSON records present in Mongo that were added since the remove changes went live.